api/build-tools/schema-sync.lua

```diff
@@ -78,6 +78,9 @@ ngx.timer = {}
 ngx.location = {}
 ngx.socket = {}
 ngx.re.gmatch = empty_function
+ngx.shared = {
+    ["plugin-api-breaker"] = {}
```

Comment:
we can use `__index` to mock all of the share dicts.

Reply:
I have submitted an issue: https://github.com/apache/apisix-dashboard/issues/671, we can fix this later.

Reply:
got it~!